.gitattributes

```diff
@@ -0,0 +1,7 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
```

Comment:
- [ ] It would be better if we highlight the piece of code we are proud of in the video presentation. Yes, you made mention of what you are proud of in the UI but could not highlight the `piece of code`

![Screenshot from 2022-08-05 07-28-51](https://user-images.githubusercontent.com/46388113/183027540-a8c3c908-3353-416f-a200-1f3ad2329b6d.png)